### DISCUSSION TITLE
Use of pre-calibrated, user-supplied realizations to help IES?

### DISCUSSION THREAD

**Discussion by sjepsen395** [usgs/pestpp#311] [Q&A] [answered]
All,

I wonder if I could get similar objective function lowering from IES as I do from PEST_HP if I provide the right user-supplied realizations for IES. I could run PEST_HP with all parameter tied (spatial uniformity) and then use the resulting calibrated parameters to define the centers of user-supplied realizations.

Does anyone have ideas of good ways to define the parameter ranges about those centers of user-supplied realizations?

I tried using posterior variances from PREDUNC7, but the resulting ranges for the more sensitive parameters looked too small to go through with. I suspect that those variances reflect spatial averages and leave out the component of variance arising from heterogeneity.

Just throwing entire parameter ranges at IES doesn't seem to work that well without some type of "pre-calibration." But I acknowledge to not having tried all possible combinations of settings.

Steve

**Comment by jtwhite79** [comment]
I think what you are missing is the correlation between parameters that the posterior bayes-linear covariance matrix from predunc7 provides.  If you generate realizations from that matrix that are centered on the calibrated par values, I bet those realizations would preform pretty well, depending on how nonlinear your inverse/inference problem is....

**Reply by sjepsen395** [reply]
Hi jtwhite79,

Good point. I was leaving that part out. However, the pre-calibration result that I operated predunc7 on did not have any spatial variation in parameters (all tied together). So the covariances of those results are with respect to spatial averages of parameters (effectively). Using those covariances to develop input ranges for realizations doesn't make that much sense to me. At least, I don't think I could defend it. I thought of a different idea--to do a "pre-calibration" in IES, take the calibrated base realization and use that as my initial parameters in PEST_HP. Unfortunately I lost all of my output results from that run because file sizes exceeded file handling timeout on Azure Batch (today, I posted a new question about turning off file outputs).

Thanks,
Steve

**Reply by jtwhite79** [reply, marked as answer]
ok.  In that case, I might be more tempted to ignore the predunc7 ranges bc they will likely be overly narrow (maybe even biased) - are any of these par values at/near a parameter bound?  If so, this can goof up the jco approximation and slow ies down...So maybe just having the final par values from pest_hp as the starting values/prior mean for ies is about the best you can do...

**Reply by sjepsen395** [reply]
Jtwhite79,

Yes, some of the pars are near a parameter bound. These have been problematic. The only solution I came up, when not using PREDUNC7, with was to back off those parameters from bounds by a somewhat arbitrary fraction of their original range. For example, not to allow any parameters closer to a bound than 25% of the entire range width. That would back off some parameters by 25% of the original range width. I'm not crazy about that because it doesn't take into account parameter sensitivity. Now I'm thinking maybe it would work better to do this all in reverse. That is, run IES first with initial parameters centered in their ranges, then use the calibrated parameters as the starting point for a PEST_HP run. That approach would use IES to compute the (hopefully) optimal initial guess of parameters for a PEST_HP run. I realize this may not give me anything better than using PEST_HP or IES on its own.

Thanks,
Steve
